### PR TITLE
chore: update golangci-lint to v1.31

### PR DESCRIPTION
**Pull request by grace0925**
Signed-off-by: Grace Liu <grace.liu@securekey.com>

### CODE REVIEW

File: cmd/issuer-rest/main_test.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 // Correct behaviour is for main to finish with exit code 0.
 // This test fails otherwise. However, this can't be checked by the unit test framework. The *testing.T argument is
 // only there so that this test gets picked up by the framework but otherwise we don't need it.
-func TestWithoutUserAgs(t *testing.T) { //nolint - see above
+func TestWithoutUserAgs(t *testing.T) { //nolint
```
Comment (fqutishat) [inline cmd/issuer-rest/main_test.go:16]: remove //nolint and change t to _

File: cmd/issuer-rest/startcmd/start.go [hunk 2]
```diff
@@ -323,7 +323,7 @@ func getRequestTokens(cmd *cobra.Command) (map[string]string, error) {
 	for _, token := range requestTokens {
 		split := strings.Split(token, "=")
 		switch len(split) {
-		case 2:
+		case 2: //nolint:gomnd
```
Comment (fqutishat) [inline cmd/issuer-rest/startcmd/start.go:326]: remove nolint

File: cmd/issuer-rest/startcmd/start.go [hunk 3]
```diff
@@ -492,8 +492,9 @@ func getOAuth2Config(cmd *cobra.Command) (*oauth2.Config, error) {
 	}
 
 	hydra := oauth2.Endpoint{
-		AuthURL:   strings.TrimSpace(authURL),
-		TokenURL:  strings.TrimSpace(tokenURL),
+		AuthURL:  strings.TrimSpace(authURL),
+		TokenURL: strings.TrimSpace(tokenURL),
```
Comment (fqutishat) [inline cmd/issuer-rest/startcmd/start.go:496]: remove nolint

File: cmd/rp-rest/main_test.go [hunk 4]
```diff
@@ -13,7 +13,7 @@ import (
 // Correct behaviour is for main to finish with exit code 0.
 // This test fails otherwise. However, this can't be checked by the unit test framework. The *testing.T argument is
 // only there so that this test gets picked up by the framework but otherwise we don't need it.
-func TestWithoutUserAgs(t *testing.T) { //nolint - see above
+func TestWithoutUserAgs(t *testing.T) { //nolint
```
Comment (fqutishat) [inline cmd/rp-rest/main_test.go:16]: remove //nolint and replace t with _

File: cmd/rp-rest/startcmd/start.go [hunk 5]
```diff
@@ -242,7 +242,7 @@ func getRequestTokens(cmd *cobra.Command) (map[string]string, error) {
 	for _, token := range requestTokens {
 		split := strings.Split(token, "=")
 		switch len(split) {
-		case 2:
+		case 2: //nolint:gomnd
```
Comment (fqutishat) [inline cmd/rp-rest/startcmd/start.go:245]: remove //nolint

Comment (grace0925) [reply cmd/rp-rest/startcmd/start.go]: done

File: cmd/rp-rest/main_test.go (hunk 4 above)
Comment (grace0925) [reply cmd/rp-rest/main_test.go]: done

File: cmd/issuer-rest/startcmd/start.go (hunk 3 above)
Comment (grace0925) [reply cmd/issuer-rest/startcmd/start.go]: done

File: cmd/issuer-rest/startcmd/start.go (hunk 2 above)
Comment (grace0925) [reply cmd/issuer-rest/startcmd/start.go]: done

File: cmd/issuer-rest/main_test.go (hunk 1 above)
Comment (grace0925) [reply cmd/issuer-rest/main_test.go]: done

**Review by rolsonquadras — APPROVED** [review]

**Review by fqutishat — APPROVED** [review]

**Pull request merged**